src/components/vuestic-mixins/SelectableList/SelectableListMixin.js

```diff
@@ -0,0 +1,66 @@
+import { FormComponentMixin } from '../FormComponent/FormComponentMixin'
+
+//Maybe we need to add some kind of fallback value to these helper methods
+//And to extract some in some kind of utility file 'cause I suppose they 
```

Comment:
Yeah. We have `utils` file. Please move it there + provide some test coverage.
Is this taken from vuetify? If yes - we can adapt their tests?